src/main/java/com/xebialabs/overthere/cifs/WinrmHttpsViaJumpstationHostnameVerifier.java

```diff
@@ -0,0 +1,19 @@
+package com.xebialabs.overthere.cifs;
+
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.SSLSession;
+
+public class WinrmHttpsViaJumpstationHostnameVerifier implements HostnameVerifier {
```

Comment:
how about naming something like JumpstationAwareHostnameVerifier to keep it generic & convey the intent

Reply:
Done